box/ansible/roles/devbox/defaults/main.yaml

```
@@ -31,6 +32,12 @@ devbox_hashicorp_packer_version: 1.8.0
 devbox_fzf_version: 0.29.0
 
 devbox_ttyd_version: 1.6.3
+devbox_ttyd_path: /usr/local/bin/tty
```

```suggestion
devbox_ttyd_path: /usr/local/bin/ttyd
```
Typo in binary name.